src/sentry/api/helpers/group_index/delete.py

```diff
@@ -85,14 +86,30 @@ def delete_group_list(
     GroupInbox.objects.filter(project_id=project.id, group_id__in=group_ids).delete()
 
     # Schedule a task per GROUP_CHUNK_SIZE batch of groups
-    for i in range(0, len(group_ids), GROUP_CHUNK_SIZE):
+    schedule_group_deletion_tasks(project.id, group_ids, transaction_id)
```

Comment:
Should we do anything with the number of scheduled tasks that comes back from this function? e.g. log or something else?

Reply:
We track it as a metric here:
https://github.com/getsentry/sentry/blob/ec18d87b04a8833b65c87cbe722c8f9d2b945970/src/sentry/tasks/delete_pending_groups.py#L73